triton-client/src/components/ProjectDiskUsage/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { FunctionComponent, useEffect } from "react"
+import { selectConstants } from "@store/constants"
+import { useAppDispatch, useAppSelector } from "@store/hooks"
+import { fetchConstants } from "@store/thunks"
+import { Col, Flex, Layout, Progress, Row } from "antd"
+import { SUPPORTED_DOWNLOAD_TYPES } from "../../constants"
+import { dataSize } from "../../functions"
```

Comment:
I wonder if we should put these codes in a specific folder 🤔 
Maybe in a folder called `@shared`, `@utils`, `@tools`?

Reply:
i think a short discussion could be good for this change